src/components/Avatar/Avatar.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+    import type { Action } from "svelte/action";
+    
+    // Note: consider `./types.ts`
+    interface AvatarProps {
+        // initials
+        initials?: string;
+        initialsFill?: string;
```

Comment:
What do you think about using `SVG` instead of `initials`? I'm kind of on the fence about either. They both have pros and cons.